presto-product-tests/bin/run_on_docker.sh

```diff
@@ -174,8 +174,15 @@ elif [[ "$ENVIRONMENT" == "multinode-tls" ]]; then
 fi
 
 CLI_ARGUMENTS="--server presto-master:8080"
-if [[ "$ENVIRONMENT" == "multinode-tls" ]]; then
-    CLI_ARGUMENTS="--server https://presto-master.docker.cluster:7778 --keystore-path /docker/volumes/conf/presto/etc/docker.cluster.jks --keystore-password 123456"
+CLI_KEYSTORE_ARGS="--keystore-path /docker/volumes/conf/presto/etc/docker.cluster.jks --keystore-password 123456"
```

Comment:
I would inline, even though this is repeated twice. Otherwise the scope of this variable is unknown